mapper.py

```diff
@@ -0,0 +1,275 @@
+    columns['Product 3 - Name'] = 'Product 3 - Name'
+    columns['Product 3 - Description'] = 'Product 3 - Description'
+    columns['Product 3 - Image Hash'] = 'Product 3 - Image Hash'
+    columns['Product 3 - Image Crops'] = None
+    columns['Product 3 - Video ID'] = None
+    columns['Product 3 - Mobile App Deep Link'] = None
+    columns['Product 4 - Link'] = None
+    columns['Product 4 - Name'] = None
+    columns['Product 4 - Description'] = None
+    columns['Product 4 - Image Hash'] = None
+    columns['Product 4 - Image Crops'] = None
+    columns['Product 4 - Video ID'] = None
+    columns['Product 4 - Mobile App Deep Link'] = None
+    columns['Product 5 - Link'] = None
+    columns['Product 5 - Name'] = None
+    columns['Product 5 - Description'] = None
+    columns['Product 5 - Image Hash'] = None
+    columns['Product 5 - Image Crops'] = None
+    columns['Product 5 - Video ID'] = None
+    columns['Product 5 - Mobile App Deep Link'] = None
+    columns['Product 6 - Link'] = None
+    columns['Product 6 - Name'] = None
+    columns['Product 6 - Description'] = None
+    columns['Product 6 - Image Hash'] = None
+    columns['Product 6 - Image Crops'] = None
+    columns['Product 6 - Video ID'] = None
+    columns['Product 6 - Mobile App Deep Link'] = None
+    columns['Product 7 - Link'] = None
+    columns['Product 7 - Name'] = None
+    columns['Product 7 - Description'] = None
+    columns['Product 7 - Image Hash'] = None
+    columns['Product 7 - Image Crops'] = None
+    columns['Product 7 - Video ID'] = None
+    columns['Product 7 - Mobile App Deep Link'] = None
+    columns['Product 8 - Link'] = None
+    columns['Product 8 - Name'] = None
+    columns['Product 8 - Description'] = None
+    columns['Product 8 - Image Hash'] = None
+    columns['Product 8 - Image Crops'] = None
+    columns['Product 8 - Video ID'] = None
+    columns['Product 8 - Mobile App Deep Link'] = None
+    columns['Product 9 - Link'] = None
+    columns['Product 9 - Name'] = None
+    columns['Product 9 - Description'] = None
+    columns['Product 9 - Image Hash'] = None
+    columns['Product 9 - Image Crops'] = None
+    columns['Product 9 - Video ID'] = None
+    columns['Product 9 - Mobile App Deep Link'] = None
+    columns['Product 10 - Link'] = None
+    columns['Product 10 - Name'] = None
+    columns['Product 10 - Description'] = None
+    columns['Product 10 - Image Hash'] = None
+    columns['Product 10 - Image Crops'] = None
+    columns['Product 10 - Video ID'] = None
+    columns['Product 10 - Mobile App Deep Link'] = None
+    columns['Image'] = 'Image'
+
+    return columns
+
+def map_file(filepath):
+    print('Mapping file found here: %s' % filepath)
+    new_columns = get_new_columns()
+    new_data = []
+    with open(filepath, 'rU') as f:
+        csv_reader = csv.DictReader(f)
+
+        for row in csv_reader:
+            result = {}
+            for col in new_columns:
+                if col == 'Campaign ID':
```

Comment:
This logic should probably be moved into a `bastardize_data` method. That way we can not bastardize it if none is allowed.